### ISSUE TITLE
[Suggestion] AI fighting each other

### ISSUE THREAD

**Issue by SabreMania** [tsunamayo/Starship-EVO#1841]
So, again, i'm not sure if this is a bug as much as anything else, but currently any AI that you spawn that's 'Hostile' will be quite literally hostile to everyone, including other Hostile AIs. I don't know how hard it would be to code for all 'Hostile' type AIs to not try to kill each other, but in any case it is pretty hard to judge the strength of a ship if it crushes other ships and can never face more than one opponement at once (because they are too busy trying to kill each other to try and kill you)

**Comment by tsunamayo** [comment]
Yes the model is simplistic for now, I will add more factions type and a deeper target selection mechanism. But I also definitely wants AI vs AI combat.
